PlatformIO/settings.ini.example

```diff
@@ -2,7 +2,7 @@
 hostname=192.168.43.140
 deployhost=192.168.43.24
 siteId=satellite
-;supported: M5ATOMECHO=0, MATRIXVOICE=1, AUDIOKIT=2, INMP441=3, INMP441MAX98357A=4, ESP32_POE_ISO=5, TAUDIO=6
+;supported: M5ATOMECHO=0, MATRIXVOICE=1, AUDIOKIT=2, INMP441=3, INMP441MAX98357A=4, ESP32_POE_ISO=5, TAUDIO=6 INMP441MAX98357AFASTLED=7
 device_type=4
```

Comment:
comma, I know both are comments but better to fix it straight away

Reply:
fixed